pkg/state/builder.go

```diff
@@ -416,3 +449,40 @@ func GetKonnectState(rawKong *utils.KongRawState,
 	}
 	return kongState, nil
 }
+
+func buildDegraphqlRouteFromCustomEntity(kongState *KongState, entity map[string]interface{}) (DegraphqlRoute, error) {
+	var degraphqlRoute DegraphqlRoute
+
+	if entity["id"] != nil {
+		degraphqlRoute.ID = kong.String(entity["id"].(string))
```

Comment:
In this function, please handle cases in which the type assertions fail. We never know what users can include in their files and failing with a graceful error is always desirable